apps/backend/src/deposits/entities/cash-deposit.entity.ts

```diff
@@ -100,6 +98,13 @@ export class CashDepositEntity {
   @Column({ name: 'file_uploaded', nullable: true })
   fileUploadedEntityId?: string;
 
+  @ManyToOne(() => BankLocationEntity, (pd) => pd.id)
+  @JoinColumn({
+    name: 'pt_location_id',
+    referencedColumnName: 'id',
+  })
+  pt_location_id: Relation<BankLocationEntity>;
```

Comment:
hrm, actually I think I will make a ticket to change this after if it's okay? This PR is so big and I will need to write a migration to rename the column in the cash deposit table for the existing columns. I am 100% on board with the name change but I think it will be easier on a separate branch after this stuff is merged in.

Reply:
Sounds fine :)